BillNye.py

```diff
@@ -0,0 +1,161 @@
+import pygame, sys
+from pygame.locals import *
+
+pygame.init()
+
+WIDTH = 1400
+HEIGHT = 932
+DISPLAY = pygame.display.set_mode((WIDTH, HEIGHT), 0, 32)
+pygame.display.set_caption('Bill Nye')
+clock = pygame.time.Clock()
+
+BLACK = (0, 0, 0)
+WHITE= (255, 255, 255) # red, green, blue in 8-bits
+RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+BLUE = (0, 0, 255)
+Delay = 200
+colourImage = pygame.image.load('images\BillNyePlus.jpg')
+colourImage = pygame.transform.scale(colourImage, (1400, 932))

```

Comment:
These [magic numbers](http://wiki.c2.com/?MagicNumber) should be refactored out of the program. For example, these might refer to `WIDTH` and `HEIGHT` respectively.